doc: add doc for package manger release workflows (#321)
oras-project/oras-www: docs/community/developer_guide.mdx

1. Click "Publish Release" button to save. Double-check that the release contains a corresponding `.asc` file for each release artifact.
1. Consume beverage of choice.. you're done! Thanks for moving the project forward.
1. Oh yea, tell people about it in the `#oras` slack channel.

### Updating package managers
#### Snap
1. Go to https://github.com/oras-project/oras/actions/workflows/release-snap.yml
1. Click on the top right drop-down `Run workflow`
  - Select a branch or tag to run the release. **This is not the actual to-be-released version** but only configuration of Github actions and Snapcraft.

>   - Enter the released version. **DO add the `v`**.

Reviewer: ```suggestion
  - Enter the release version. **The release should start with a `v`**.
```

I was not entirely sure if DO was capitalized for emphasis or typo.

>   - (optional) Check the checkbox only when the target version is stable.

Reviewer: ```suggestion
  - If the target version is stable, check the stable release checkbox.
```
